### PR TITLE
Communication: move Bullhorn to Forum list

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Communication: move Bullhorn to Forum list

### CODE REVIEW

**Comment by Andersson007** [comment]
I think @oraNod 's suggestion about moving Bullhorn to the forum list is fair that it's now a part of the forum.
cc @felixfontein and everyone else who see this:)

**Comment by felixfontein** [comment]
I'm a bit hesitant since the list is already pretty long, and the Bullhorn is already covered under News & Announcements.

**Comment by Andersson007** [comment]
> I'm a bit hesitant since the list is already pretty long, and the Bullhorn is already covered under News & Announcements.

Should we maybe remove news and announcements then? Folks will find it on their own if they need as there are refs in the communication guide anyway. I think it's worth highlighting the bullhorn anyway.

File: README.md [hunk 1]
```diff
@@ -31,13 +31,12 @@ If your collection is not present on the Ansible forum yet, please check out the
...
-
-* The Ansible [Bullhorn newsletter](https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn): used to announce releases and important changes.
+  * [Get Help](https://forum.ansible.com/c/help/6): get help or help others. Please add appropriate tags if you start new discussions, for example the `YOUR TAG` tag.
+  * [Posts tagged with 'your tag'](https://forum.ansible.com/tag/YOUR_TAG): subscribe to participate in collection/technology-related conversations.
+  * [Refer to your forum group here if exists](https://forum.ansible.com/g/): by joining the team you will automatically get subscribed to the posts tagged with [your group forum tag here](https://forum.ansible.com/tags).
+  * [Bullhorn newsletter](https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn): used to announce releases and important changes.
+  * [Social Spaces](https://forum.ansible.com/c/chat/4): gather and interact with fellow enthusiasts.
+  * [News & Announcements](https://forum.ansible.com/c/news/5): track project-wide announcements including social events.
```
Comment (felixfontein) [inline README.md:39]:
```suggestion
  * [News & Announcements](https://forum.ansible.com/c/news/5): track project-wide announcements including social events. The [Bullhorn newsletter](https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn) can also be found here.
```

**Review by felixfontein — COMMENTED** [review]
I would just keep News and Announcements. The Bullhorn is very easy to find there: 
image

How about simply mentioning the Bullhorn in the News and Announcements line?

**Comment by Andersson007** [comment]
@felixfontein SGTM, committed and updated it slightly, PTAL

**Review by felixfontein — APPROVED** [review]

**Comment by Andersson007** [comment]
@felixfontein thanks for reviewing!

**Pull request merged**